src/endpoint/endpoint.js

```diff
@@ -349,6 +355,21 @@ function fork_count_handler(req, res) {
     res.end(JSON.stringify(reply));
 }
 
+/**
+ * version_handler return the version of noobaa package
+ * @param {*} req 
+ * @param {*} res 
```

Comment:
Use these types taken from the EndpointHandler definition -
```suggestion
 *  @param {EndpointRequest} req
 *  @param {import('http').ServerResponse} res
```